cpp/src/ns/ns_evolution.cpp

```diff
@@ -471,6 +499,57 @@ double FreddiNeutronStarEvolution::R_Alfven() const {
 	return ns_str_->args_ns.R_Alfven(GM(), Mdot_in());
 }
 
+double FreddiNeutronStarEvolution::R_Alfven_basic() const {
+	return ns_str_->args_ns.R_Alfven_basic(GM(), Mdot_in());
+}//надо переписать это так же, как mu_magn
+
+
+
+double FreddiNeutronStarEvolution::R_Magn_bozzo18() const {
+	const double chi_oblique_rad = double(chioblique()) * M_PI / 180.;
```

Comment:
Why do you need this `double()` casting here?

Reply:
done